#316: Specify GCS credentials via CONNECTION (#317)
exasol/cloud-storage-extension: doc/changes/changes_2.8.0.md

# Cloud Storage Extension 2.8.0, released 2024-05-17

Code name: Simplify GCS Configuration

## Summary

> This release allows configuring Google Cloud Storage (GCS) via a `CONNECTION` instead of uploading the credentials JSON file to BucketFS. This does not expose GCP credentials in BucketFS and it is easier to configure. See the [user guide](../user_guide/user_guide.md#configure-gcp-credentials) for details.

Reviewer: ```suggestion
This release allows configuring Google Cloud Storage (GCS) via a `CONNECTION` instead of uploading the credentials JSON file to BucketFS. This avoids exposing GCP credentials as file in BucketFS and simplifies configuration. See the [user guide](../user_guide/user_guide.md#configure-gcp-credentials) for details.
```

> 

Reviewer: ```suggestion
Please note for backwards compatibility you can still provide the GCS credentials as a file although CSE recommends configuring Google Cloud Storage (GCS) via a `CONNECTION`.

```
